recipes/lcaclassifier/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Looks like the software is meant to be installed with `install.sh` instead of using `setup.py` directly.

Reply:
I will need `buildout`/`zc.buildout' in that case, so, I will add package for it to conda-forge maybe. Thanks.